libs/common/src/key-management/crypto/services/web-crypto-function.service.ts

```diff
@@ -1,6 +1,8 @@
 import * as argon2 from "argon2-browser";
 import * as forge from "node-forge";
 
+import { EncryptionType } from "@bitwarden/common/platform/enums";
```

Comment:
```suggestion
import { EncryptionType } from "../../../platform/enums";
```
Relative imports within the same package.